### PR TITLE
Add folder user and file icons

### PR DESCRIPTION
**Pull request by davidzzz**

### CODE REVIEW

File: src/@next/Icon/icons/icons.ts [hunk 1]
```diff
@@ -59,6 +59,8 @@ export const iconNames = [
   'ri-facebook-circle-line',
   'ri-file-fill',
   'ri-file-line',
+  'ri-file-list-3',
```
Comment (ninariccimarie) [inline src/@next/Icon/icons/icons.ts]:
```suggestion
  'ri-file-list3-fill',
```

can we use this naming instead to follow convention. we'll try to improve documentation soon to help to guide with the naming format 🙇

File: src/@next/Icon/icons/icons.ts [hunk 2]
```diff
@@ -59,6 +59,8 @@ export const iconNames = [
...
+  'ri-folder-user',
```
Comment (ninariccimarie) [inline src/@next/Icon/icons/icons.ts]:
```suggestion
  'ri-folder-user-fill',
```

same with this one 🙇

File: src/@next/Icon/icons/ri-file-list-3.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 18 17" xmlns="http://www.w3.org/2000/svg">
+<path d="M14.8333 16.8333H3.16663C2.50358 16.8333 1.8677 16.5699 1.39886 16.1011C0.930018 15.6323 0.666626 14.9964 0.666626 14.3333V0.999999C0.666626 0.778986 0.754423 0.567024 0.910704 0.410744C1.06698 0.254463 1.27895 0.166666 1.49996 0.166666H13.1666C13.3876 0.166666 13.5996 0.254463 13.7559 0.410744C13.9122 0.567024 14 0.778986 14 0.999999V11H17.3333V14.3333C17.3333 14.9964 17.0699 15.6323 16.6011 16.1011C16.1322 16.5699 15.4963 16.8333 14.8333 16.8333ZM14 12.6667V14.3333C14 14.5543 14.0878 14.7663 14.244 14.9226C14.4003 15.0789 14.6123 15.1667 14.8333 15.1667C15.0543 15.1667 15.2663 15.0789 15.4225 14.9226C15.5788 14.7663 15.6666 14.5543 15.6666 14.3333V12.6667H14ZM3.99996 4.33333V6H10.6666V4.33333H3.99996ZM3.99996 7.66667V9.33333H10.6666V7.66667H3.99996ZM3.99996 11V12.6667H8.16663V11H3.99996Z" fill="#666666"/>
```
Comment (ninariccimarie) [inline src/@next/Icon/icons/ri-file-list-3.svg]:
```suggestion
<path d="M14.8333 16.8333H3.16663C2.50358 16.8333 1.8677 16.5699 1.39886 16.1011C0.930018 15.6323 0.666626 14.9964 0.666626 14.3333V0.999999C0.666626 0.778986 0.754423 0.567024 0.910704 0.410744C1.06698 0.254463 1.27895 0.166666 1.49996 0.166666H13.1666C13.3876 0.166666 13.5996 0.254463 13.7559 0.410744C13.9122 0.567024 14 0.778986 14 0.999999V11H17.3333V14.3333C17.3333 14.9964 17.0699 15.6323 16.6011 16.1011C16.1322 16.5699 15.4963 16.8333 14.8333 16.8333ZM14 12.6667V14.3333C14 14.5543 14.0878 14.7663 14.244 14.9226C14.4003 15.0789 14.6123 15.1667 14.8333 15.1667C15.0543 15.1667 15.2663 15.0789 15.4225 14.9226C15.5788 14.7663 15.6666 14.5543 15.6666 14.3333V12.6667H14ZM3.99996 4.33333V6H10.6666V4.33333H3.99996ZM3.99996 7.66667V9.33333H10.6666V7.66667H3.99996ZM3.99996 11V12.6667H8.16663V11H3.99996Z" />
```
always make sure to remove the fill. one indicator is it should show in black on preview

File: src/@next/Icon/icons/ri-folder-user.svg [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 18 15" xmlns="http://www.w3.org/2000/svg">
+<path d="M9.34496 1.66667H16.5C16.721 1.66667 16.9329 1.75446 17.0892 1.91074C17.2455 2.06702 17.3333 2.27899 17.3333 2.5V14.1667C17.3333 14.3877 17.2455 14.5996 17.0892 14.7559C16.9329 14.9122 16.721 15 16.5 15H1.49996C1.27895 15 1.06698 14.9122 0.910704 14.7559C0.754423 14.5996 0.666626 14.3877 0.666626 14.1667V0.833333C0.666626 0.61232 0.754423 0.400358 0.910704 0.244078C1.06698 0.0877973 1.27895 0 1.49996 0H7.67829L9.34496 1.66667ZM8.99996 8.33333C9.55249 8.33333 10.0824 8.11384 10.4731 7.72314C10.8638 7.33244 11.0833 6.80253 11.0833 6.25C11.0833 5.69747 10.8638 5.16756 10.4731 4.77686C10.0824 4.38616 9.55249 4.16667 8.99996 4.16667C8.44743 4.16667 7.91752 4.38616 7.52682 4.77686C7.13612 5.16756 6.91663 5.69747 6.91663 6.25C6.91663 6.80253 7.13612 7.33244 7.52682 7.72314C7.91752 8.11384 8.44743 8.33333 8.99996 8.33333ZM5.66663 12.5H12.3333C12.3333 11.6159 11.9821 10.7681 11.357 10.143C10.7319 9.51786 9.88402 9.16667 8.99996 9.16667C8.1159 9.16667 7.26806 9.51786 6.64294 10.143C6.01782 10.7681 5.66663 11.6159 5.66663 12.5Z" fill="#666666"/>
```
Comment (ninariccimarie) [inline src/@next/Icon/icons/ri-folder-user.svg]:
```suggestion
<path d="M9.34496 1.66667H16.5C16.721 1.66667 16.9329 1.75446 17.0892 1.91074C17.2455 2.06702 17.3333 2.27899 17.3333 2.5V14.1667C17.3333 14.3877 17.2455 14.5996 17.0892 14.7559C16.9329 14.9122 16.721 15 16.5 15H1.49996C1.27895 15 1.06698 14.9122 0.910704 14.7559C0.754423 14.5996 0.666626 14.3877 0.666626 14.1667V0.833333C0.666626 0.61232 0.754423 0.400358 0.910704 0.244078C1.06698 0.0877973 1.27895 0 1.49996 0H7.67829L9.34496 1.66667ZM8.99996 8.33333C9.55249 8.33333 10.0824 8.11384 10.4731 7.72314C10.8638 7.33244 11.0833 6.80253 11.0833 6.25C11.0833 5.69747 10.8638 5.16756 10.4731 4.77686C10.0824 4.38616 9.55249 4.16667 8.99996 4.16667C8.44743 4.16667 7.91752 4.38616 7.52682 4.77686C7.13612 5.16756 6.91663 5.69747 6.91663 6.25C6.91663 6.80253 7.13612 7.33244 7.52682 7.72314C7.91752 8.11384 8.44743 8.33333 8.99996 8.33333ZM5.66663 12.5H12.3333C12.3333 11.6159 11.9821 10.7681 11.357 10.143C10.7319 9.51786 9.88402 9.16667 8.99996 9.16667C8.1159 9.16667 7.26806 9.51786 6.64294 10.143C6.01782 10.7681 5.66663 11.6159 5.66663 12.5Z" />
```
same here as well

File: src/@next/Icon/icons/ri-file-list-3.svg (hunk 3 above)
Comment (davidzzz) [reply src/@next/Icon/icons/ri-file-list-3.svg]: ah, okay. I was just look up to svg tag only, will take case of path tag as well

**Review by ninariccimarie — CHANGES_REQUESTED** [review]
@davidzzz sorry I just realised the exported file was not exported from the "group" layer. 😓 
it will actually have imbalanced resizing if ever. 

can you export the group layer instead?
Screenshot 2023-05-30 at 4 48 36 PM

**Review by ninariccimarie — APPROVED** [review]
LGTM 🚀

**Pull request merged**